pgrx/src/pg_catalog/pg_proc.rs

```diff
@@ -218,12 +218,10 @@ impl PgProc {
         self.get_attr(pg_sys::Anum_pg_proc_proconfig)
     }
 
-    /// ```text
-    /// An array of the modes of the function arguments, encoded as i for IN arguments, o for OUT
-    /// arguments, b for INOUT arguments, v for VARIADIC arguments, t for TABLE arguments. If all
-    /// the arguments are IN arguments, this field will be null. Note that subscripts correspond to
-    /// positions of proallargtypes not proargtypes.
-    /// ```
+    /// > An array of the modes of the function arguments, encoded as i for IN arguments, o for OUT
+    /// > arguments, b for INOUT arguments, v for VARIADIC arguments, t for TABLE arguments. If all
+    /// > the arguments are IN arguments, this field will be null. Note that subscripts correspond to
+    /// > positions of proallargtypes not proargtypes.
```

Comment:
Hmm. Reviewing this, it's not clear what this is even quoting or the context.

Reply:
Oh, it's from https://www.postgresql.org/docs/current/catalog-pg-proc.html

Can you add a link to that somewhere in this type's docs?

Reply:
let me know how it should be reworded. Perhaps just use a `suggest` github feature?

Reply:
thx, added a link